src/style.css

```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0,0,0,0.05);
+}
+
+.logo__link {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.logo__img {
+  width: 100%;
+  height: 100%;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__item {
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  font-family: Roboto, sans-serif;
+  display: flex;
+  align-items: center;
+  height: 60px;
+  white-space: nowrap;
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+  transition: color 0.3s;
+  font-size: 12px;
+  position: relative;
+  font-weight: 500;
+}
+
+.nav__link:hover,
+a.is-active {
+  color: #00acdc;
+}
+
+a.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  display: block;
+  background-color: #00acdc;
```

Comment:
better to move all repeatable colors to some vars

Reply:
all done, I hope it`s correct now